histomicstk/features/ComputeIntensityFeatures.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pandas as pd
+import scipy.stats
+from skimage.measure import regionprops
+
+
+def ComputeIntensityFeatures(im_label, im_intensity,
+                             num_hist_bins=10, rprops=None):
+    """
+    Calculates intensity features from an intensity image.
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    im_intensity : array_like
+        Intensity image
+
+    num_hist_bins: int, optional
+        Number of bins used to computed the intensity histogram of an object.
+        Histogram is used to energy and entropy features. Default is 10.
```

Comment:
@cooperlab @slee172 Should we set the number of bins using something like [this](http://stats.stackexchange.com/a/862)